optuna/artifacts/_download.py

```diff
@@ -18,6 +18,11 @@ def download_artifact(artifact_store: ArtifactStore, artifact_id: str, file_path
             The identifier of the artifact to download.
         file_path:
             A path to save the downloaded artifact.
+
+    .. note::
+        Optuna does not provide any API to save artifact store information linked to each artifact,
```

Comment:
The phrase `artifact store information` might be a bit ambiguous. How about providing a specific example to clarify?